core/src/main/kotlin/db/UserDao.kt

```diff
@@ -34,6 +34,9 @@ interface UserDao {
     @Query("UPDATE users SET status = :status WHERE public_key == :publicKey")
     fun updateStatus(publicKey: String, status: UserStatus)
 
+    @Query("UPDATE contacts SET avatar_uri = :uri WHERE public_key = :publicKey")
+    fun updateAvatarUri(publicKey: String, uri: String)
```

Comment:
I don't see that this is used anywhere?

If the database change isn't used in this PR, it should be reverted. Database changes are special in that you need to also write a migration to convert the database to a version with the new data (as well as a test for it, because I don't want to lose any user data!), and if you update past a migration, you won't be able to revert to an older version, so changes to the database should only be made once they're actually needed/used. :P

Reply:
For those who read this, field will remain as discussed between us and support for user avatars will be implemented.